app/templates/services/service_submission.html

```diff
@@ -34,7 +36,17 @@
     {{ last_edit.createdAt|datetimeformat }}<br />
     by {{ last_edit.userName }}
   </p>
-  {% if service_data.status == 'not-submitted' and unanswered_questions == 0 %}
+  {% if unanswered_required or unanswered_optional %}
+    <p class="last-edited">
+      {{ submission.multiline_string(
+        submission.unanswered_required_text(unanswered_required, unanswered_optional),
+        submission.unanswered_optional_text(unanswered_required, unanswered_optional)
+      ) }}
+    </p>
+  {% endif %}
+  {% if service_data.status == 'submitted' %}
+    <span class="service-status-published">This service is complete and will be automatically submitted at 3pm&nbsp;BST, 6 October</span>
```

Comment:
Do we know this is the date, or is this a placeholder for now?  If a placeholder might be worth making it more obvious so that we definitely remember to change the message when we know the date!